### DISCUSSION TITLE
million lumen homepage

### DISCUSSION THREAD

**Discussion by vinamogit** [stellar/sorobounty-spectacular#11] [General]
Here is my still work in progress contribution, a million lumen homepage.

It features a collaborative fresco composed of 4096 NFTs, each containing 256 pixels.

It contains:
- an implementation of ERC721, demonstrating how to convert ethereum standard to soroban
- a contract that uses the ERC721 implementation
- a dApp or kind of that exposes the contract features, letting you mint and show your NFT
- a storage crate that allows more convenient api for the soroban storage access

My goal is to add features (like each owner could publish their own erc721 and manage them) and also add educational content along the way.

The demo: [https://millionlumenhomepage.art/](https://millionlumenhomepage.art/)
The code: [https://github.com/candela-network/millionlumenhomepage.art/tree/prod](https://github.com/candela-network/millionlumenhomepage.art/tree/prod)
The contract: [CBR33SLBHCH7DRYIXT4PAOI6KIU6IDHH72FMJGCIGUBKRVU6HLQ4HXVN](https://candela.network/pages/getwasm/?id=CBR33SLBHCH7DRYIXT4PAOI6KIU6IDHH72FMJGCIGUBKRVU6HLQ4HXVN#)

### READMEs
- [How to deploy on standalone](https://github.com/candela-network/millionlumenhomepage.art/blob/main/README.md#how-to-launch-the-dapp-on-standalone)
- [ERC721 implementation for Soroban](https://github.com/candela-network/millionlumenhomepage.art/tree/main/erc721#erc721-implementation-for-soroban)

### video
[Deploying a soroban dApp - millionlumenhomepage](https://youtu.be/cBb_76kqqfg)

**Comment by esteblock** [comment]
Great!

**Comment by briwylde08** [comment]
Thank you for your submission! We will review and get back to you within a week. 

Best,
Bri

**Comment by briwylde08** [comment]
Hello @vinamogit we've reviewed your submission! [🎉](https://emojipedia.org/party-popper/)

Given our guidelines, your submission may qualify for an award of $1,250 worth of XLM. To receive that award, please fill out [this form](https://stellarform.typeform.com/to/HBjEyCJt) completely and accurately.

Thanks again! Please let us know if you have any questions.

Best,
Bri

**Reply by vinamogit** [reply]
Thanks!

**Comment by fazzatti** [comment]
Congratulations @vinamogit ! This use case is awesome!

**Comment by briwylde08** [comment]
Heya @vinamogit!

Thank you for providing the requested information! You’ve passed KYC and we’ve logged your award payment of $1,250 worth of XLM, which will be sent to the wallet you designated in the next week.

Thanks again and please let me know if you have any questions!

Best,
Bri
